locales/help_enu/gkhIntro.html

```diff
@@ -1,57 +1,64 @@
-<p>.</p>
+    <h1>Introduction</h1>
+    <p>There are three principal factors the basics of Genealogy:</p>
+    <ol>
+        <li>A person&#39;s name (who?)</li>
+        <li>Location--where an event occurred (where?)</li>
+        <li>Date of an event (when?)</li>
+    </ol>
+    <p>
+        Using GEDKeeper you can process huge amounts of genealogical data.
+        The main statement here is that GEDKeeper honors these three main
+        factors. For example, you can quickly find all namesakes, or all people
+        from the same community, or everyone who lived in a specific period of
+        time. You can filter data using all the three factors. You could, for example,
+        select namesakes and who lived in the same community in the specified year.
+    </p>
+    <p>
+        You collect information (create a database) using different sources
+        (including archives of all kinds). The following are two main sources:
+        Audit tales (audits and censuses made on a more earlier or later time)
+        and registers of births. Other sources are also possible, but we take
+        the two sources as a basic.
+    </p>
+    <p>
+        A database filled with entries from sources may be counted as a
+        conditionally complete one. For example, we have data defining all
+        citizens of a specific locality at a specific period of time; if we had
+        used audits (censuses). Later we were adding more and more new
+        information, and finally we accumulated a huge data array with many people
+        living in several locations during different times. Such a database is
+        very useful for a genealogist because no one knew about relations
+        between people in the database previously. This becomes increasingly obvious
+        after inspecting and including other sources.
+    </p>
+    <p>
+        Why is a database called conditionally complete? Because it does not
+        contain strictly accurate data. Many data are approximate, or even
+        estimated. For example, a date in the database can be exact
+        (06/12/1954), or less exact (1954), or approximate
+        (around 1976), or estimated (before 1956 and after
+        1873). It is very important for software to handle such dates
+        correctly&#151;and GEDKeeper does!
+    </p>
+    <p>
+        GEDKeeper has a very effective tool for indicating stretches of time:
+        &quot;<a href="gkhSvc_TimeLine.html" target="_self">Time line</a>&quot;. This
+        tool can make time samples for events. Using Time line you
+        can create a model of the original source&#151;an archival document. For
+        example, after we have made a time sample for 1858 using
+        locations of events, we can get a list similar to the original source&#151;people
+        who lived in a specific community in 1858 (as in tenth
+        audit of 1858). This allows us to check the accuracy of all information
+        added to database.
+    </p>
```

Comment:
The file is changed almost completely, no specific change is visible - a solid "flare". It will not work, you need to correct the formatting back.

Reply:
I plan on replacing my fork completely. I think that rebase will do the trick--now I just have to figure out how to do it. Unless you care to show me the command(s)? FYI: I have not installed TortoiseGit yet--it implies changes to the VS Git support (something I want to avoid).